src/lib/dialog/dialog.scss

```diff
@@ -34,26 +32,24 @@ $mat-dialog-button-margin: 8px !default;
 
 .mat-dialog-content {
   display: block;
-  margin: 0 $mat-dialog-padding * -1;
-  padding: 0 $mat-dialog-padding;
+  margin-bottom: -8px;
```

Comment:
Can you move all of these values out into variables?

Reply:
I created StackBlizt demo, please review.